lib/ash/page/offset.ex

```diff
@@ -7,7 +7,23 @@ defmodule Ash.Page.Offset do
   """
   defstruct [:results, :limit, :offset, :count, :rerun, :more?]
 
-  @type t :: %__MODULE__{}
+  @type t :: %__MODULE__{
+          results: [Ash.Resource.t()],
```

Comment:
`Ash.Resource.record` the `t` type refers to a module

Reply:
Okay.

At what point is it a Resource.t versus Resource.record?

Reply:
A `Resource.t` is `MyApp.Accounts.User` and `Resource.record` is `%MyApp.Accounts.User{}`